pkgs/development/tools/rtags/default.nix

```diff
@@ -16,11 +16,13 @@ stdenv.mkDerivation rec {
            MACOSX_DEPLOYMENT_TARGET="10.9"
   '';
 
+  NIX_CFLAGS_COMPILE = "-fexceptions";
```

Comment:
The exact method will depend on the build system used by the package in question. In this case, it seems like we needed to work around an issue with the flags we were already passing in a very rtags-specific environment variable.